nucypher/blockchain/eth/agents.py

```diff
@@ -186,11 +191,17 @@ def approve_transfer(
         """Approve the spender address to transfer an amount of tokens on behalf of the sender address"""
         self._validate_zero_allowance(amount, spender_address, transacting_power)
 
-        payload: TxParams = {'gas': Wei(500_000)}  # TODO #842: gas needed for use with geth! <<<< Is this still open?
-        contract_function: ContractFunction = self.contract.functions.approve(spender_address, amount)
-        receipt: TxReceipt = self.blockchain.send_transaction(contract_function=contract_function,
-                                                              payload=payload,
-                                                              transacting_power=transacting_power)
+        payload: TxParams = {
+            "gas": Wei(500_000)
+        }  # TODO #842: gas needed for use with geth! <<<< Is this still open?
```

Comment:
This whole method can be removed (although not necessarily in this PR)

Reply:
You see this is an example of the linter getting me in trouble at work! 😆 I hope you don't mind if we leave this for another PR, it's out of scope.

Reply:
addressed in 8ddd452137cd37f3b0d99a04ea9b7eda65af63d1